src/engraving/libmscore/instrtemplate.cpp

```diff
@@ -306,7 +306,7 @@ void InstrumentTemplate::write(XmlWriter& xml) const
     }
     stringData.write(xml);
     if (staffCount > 1) {
-        xml.tag("staves", staffCount);
+        xml.tag("staves", static_cast<int>(staffCount));
```

Comment:
Or a new `QVariant`? But how exactly?

Reply:
It is better to use `int`, because there will be a value type check inside...

Very soon we will get rid of QVariant completely, this code will be changed

Reply:
Via your last PR #11510 that change is now in the repository anyway, as far as I can tell.